### PR TITLE
Spp 9143 set up concourse

### PR DESCRIPTION
**Pull request by jhmjam**
# Description

Initial set up of concourse pipelines which are manually triggered (will be automated in another ticket) and run the BDD tests.

This has been set up for the prod, preprod and prod environments.

Co-author: Jason Bell
Co-author: James Morgan
Co-author: Sanjeev Srithevan

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Technical Enhancements

# Checklist:

If any of these are not completed, please explain why in the notes.

## **Definition of Done**
**Code and merges**

- [x] Code to be commented on where applicable 
- [ ] Documentation updated where required 
- [x] Have considered non-functional requirements such as Security, Performance, Scalability, and Fault Tolerance
- [x] I have linted the code

**Testing**

- [x] All levels of acceptance test are passing (automated, integration, manual, accessibility, etc.)
- [x] I have run the behave command to check the selenium behaviour tests pass locally
- [x] Acceptance criteria met

**Other Checks**
- [x] I have performed a self-review of my own code
- [x] I have checked for spelling errors
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes don't break anything unexpected
- [x] I have checked and updated the security.txt file where required
- [x] Up to date with the main branch

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  - name: dev-bdd-tests
+    plan:
+      - get: main
+      - task: run-acceptance-tests
+        config:
+          platform: linux
+          image_resource:
+            type: registry-image
+            source:
+              repository: jasonbellons/python_3_slim
```
Comment (nsmithuk) [inline ci/pipeline.yml:25]: Please can we avoid using images in personal repos. I'm happy to jump on a call to work out the best image/place to host it.

**Review by nsmithuk — CHANGES_REQUESTED** [review]

File: ci/pipeline.yml (hunk 1 above)
Comment (JasonBellONS) [reply ci/pipeline.yml]: amended to use the new selenium image. confirmed it now runs on all environments with this image

File: ci/tasks/list-repo.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
Comment (nsmithuk) [inline ci/tasks/list-repo.sh:1]: Can we delete this file now?

File: ci/tasks/list-repo.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (nsmithuk) [inline ci/tasks/list-repo.yml:1]: Can we delete this file now?

File: ci/pipeline.yml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+# The triggers are manual for now, a ticket has been written to address this.
+resources:
+  - name: main
+    type: git
+    icon: github
+    check_every: never
+    webhook_token: gesture
+    source:
+      uri: https://github.com/ONSdigital/sml-catalogue.git
+      branch: SPP-9143-set-up-concourse
```
Comment (nsmithuk) [inline ci/pipeline.yml:11]: You'll probably want to set this to `main` now.

**Review by nsmithuk — CHANGES_REQUESTED** [review]

File: ci/tasks/list-repo.sh (hunk 2 above)
Comment (JasonBellONS) [reply ci/tasks/list-repo.sh]: yep, file removed

File: ci/tasks/list-repo.yml (hunk 3 above)
Comment (JasonBellONS) [reply ci/tasks/list-repo.yml]: yep, file removed

**Review by nsmithuk — APPROVED** [review]

**Pull request merged**